### ISSUE TITLE
[Question] How to uninstall

### ISSUE THREAD

**Issue by MicaelJarniac** [stijnwop/manureSystem#95]
Don't get me wrong, this seems to be an awesome mod, and I do look forward to playing with it.

But I've installed it on a save and don't actively use it right now, and I wanted to uninstall the mod.

When I disable the mod before loading the save, it never finishes loading, and I have to force-quit the game, so I wanted to know if there's a "correct" way of uninstalling it.

**Comment by stijnwop** [comment]
Can you share your log?

**Comment by MicaelJarniac** [comment]
I sure can! Sorry for not including it straight away, I didn't even think about it.

Working game, mod still active: https://pastebin.com/CUDFT8ci
Working game, mod deactivated: https://pastebin.com/gUvHZrFQ

Okay, wait a bit, I think I solved it.

Yesterday, I had two herbicide crates on my farm, and the mod was installed. When deactivating the mod, the save wouldn't load at all.

Later that day, I transferred all herbicide to tanks, so I no longer had the crates on my farm.

Today, when creating the logs to send you, I noticed the problem was gone, and my save loaded successfully with the mod deactivated.

So I believe the problem is, the mod modifies the crates to add the hose attachment. If there is a crate somewhere, the mod can't be deactivated, but if there are no crates, it can be deactivated without a problem.

I'll do some more testing, but I believe that's what happened.

**Comment by MicaelJarniac** [comment]
Well, maybe not. I created a new save with only this mod, purchased a herbicide pallet, saved and quit the save, then deactivated this mod, tried loading the save and it worked just fine.

I'm not sure what happened yesterday, or what fixed it, but I still think it's somehow related to those pallets, as I believe those were the only things that I did that could have affected it.

Maybe it was the pallets + a mod conflict, I don't know.

But anyways, sorry for this quite random issue, and thanks for the awesome mod!

I won't close the issue right now because maybe there's some clue in the logs, or maybe we're close to finding a bug, but feel free to close it if you want to.

I'm up for doing more testing if needed.
